parquet/src/main/java/org/apache/iceberg/parquet/ParquetDictionaryRowGroupFilter.java

```diff
@@ -99,8 +93,8 @@ public boolean shouldRead(MessageType fileSchema, BlockMetaData rowGroup,
     private Map<Integer, Function<Object, Object>> conversions = null;
 
     private boolean eval(MessageType fileSchema, BlockMetaData rowGroup,
-                         DictionaryPageReadStore dictionaries) {
-      this.dictionaries = dictionaries;
+                         DictionaryPageReadStore dict) {
+      this.dictionaries = dict;
```

Comment:
Nit: maybe `dictionaryReadStore` is a better name.

Reply:
I agree, thanks